internal/server/data/model/model.go

```diff
@@ -0,0 +1,17 @@
+//go:build template
+
+package model
+
+type TemplateTable struct{}
```

Comment:
Maybe a comment in here that this is used for file generation would be good

Reply:
Comments would be great! I tried to add comments, but the way `go/ast` handles them is super weird. I think it uses some kind of heuristics to decide where to place the comments, and those can get messed up when the target file doesn't have any comments. It was causing really strange output formatting.

I'll try to add some comments to the target file and try to restore comments to this template.